docs/app/docs/overlay/sonner/page.mdx

```diff
@@ -0,0 +1,132 @@
+            ),
+          ),
+        );
+
+        return FSonner(
+          child: Builder(
+            builder: (context) => Expanded(
+              child:
+               Center(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  spacing: 5,
+                  children: [
+                    FButton(
+                      intrinsicWidth: true,
+                      onPress: () {
+                        Widget buildToast(BuildContext context, FToast toast) => IntrinsicHeight(
+                          child: FCard(
+                            style: cardStyle,
+                            title: const Text('Event has been created'),
+                            subtitle: const Padding(
+                              padding: EdgeInsets.symmetric(vertical: 5),
+                              child: Text('This is a brief description that provides additional context.'),
+                            ),
+                            child: FButton(onPress: () => toast.dismiss(), child: const Text('Undo')),
+                          ),
+                        );
+
+                        showFToast(context: context, builder: buildToast);
+                      },
+                      child: const Text('Show Small Toast'),
+                    ),
+                    FButton(
+                      intrinsicWidth: true,
+                      onPress: () {
+                        Widget buildToast(BuildContext context, FToast toast) => IntrinsicHeight(
+                          child: FCard(
+                            style: cardStyle,
+                            title: const Text('Event has been created'),
+                            subtitle: const Padding(
+                              padding: EdgeInsets.symmetric(vertical: 5),
+                              child: Text(
+                                'This is a more detailed description that provides comprehensive context and additional '
+                                'information about the notification, explaining what happened and what the user might expect '
+                                'next.',
+                              ),
+                            ),
+                            child: FButton(onPress: () => toast.dismiss(), child: const Text('Undo')),
+                          ),
+                        );
+
+                        showFToast(context: context, builder: buildToast);
+                      },
+                      child: const Text('Show Large Toast'),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+        );
+      }
+    }
+    ```
+
+  </Tabs.Tab>
+</Tabs>
+
+## CLI
+
+To generate and customize this style:
+
+```shell copy
+dart run forui style create sonner 
+```
+
+## Usage
+
+### `showFToast(...)`
+
+```dart copy
+showFToast(
+  context: context,
+  style: FSonnerStyle(...),
```

Comment:
I noticed that we're quite inconsistent with the style parameter. Should we include it?

Reply:
Please don't resolve the issue if it isn't resolved. It's kinda hard to track. Should we file an issue?

Reply:
Filed under https://github.com/forus-labs/forui/issues/543